coss/chain.json

```diff
@@ -0,0 +1,117 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "coss",
+  "chain_id": "coss-1",
+  "website": "https://coss.ink/",
+  "pretty_name": "COSS",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "coss",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md